app/controllers/orders_controller.rb

```diff
@@ -37,13 +37,19 @@ def create
         project_id: project.id,
         order_id: @order.id,
         quantity: quantity,
-        line_project_cost: project.price * quantity.to_i
+        line_item_cost: project.price * quantity.to_i
         )
     end
     session.delete(:cart)
     redirect_to user_order_path(user_id: session[:user_id], id: @order.id)
   end
 
+  def update_project_quantity
+    session[:cart][params[:update_project_quantity][:project_id]] =
+      params[:update_project_quantity][:quantity]
+    flash[:notice] = "Project quantity updated"
```

Comment:
Orion and I changed this method. We can merge it in and orion and I can submit our PR.